packages/lib/services/commands/EditorFocusUtil.ts

```diff
@@ -0,0 +1,31 @@
+import CommandService from '../CommandService';
+
+// Editor commands will focus the editor after they're executed
+export function isEditorCommand(commandName: string) {
```

Comment:
The file is called "EditorFocusUtils" but this command is unrelated to editor focus so please move it to isEditorCommand.ts